integration-test/1491-malaysian-road-shields.py

```diff
@@ -0,0 +1,509 @@
+                'access': u'yes',
+                'bicycle': u'yes',
+                'cycleway': u'no',
+                'fixme': u'yes',
+                'foot': u'yes',
+                'highway': u'secondary',
+                'horse': u'no',
+                'lanes': u'2',
+                'lit': u'no',
+                'motor_vehicle': u'yes',
+                'name': u'Jalan Paal-Kuala Tomani',
+                'oneway': u'no',
+                'ref': u'SA33',
+                'source': u'openstreetmap.org',
+                'surface': u'asphalt',
+            }),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 32770972,
+                'network': u'MY:SBH',
+                'shield_text': u'SA33',
+            })
+
+    def test_t11_my_trg(self):
+        import dsl
+
+        z, x, y = (16, 51497, 31821)
+
+        self.generate_fixtures(
+            dsl.is_in('MY', z, x, y),
+            # https://www.openstreetmap.org/way/32767537
+            dsl.way(32767537, dsl.tile_diagonal(z, x, y), {
+                'highway': u'secondary',
+                'oneway': u'no',
+                'ref': u'T11',
+                'source': u'openstreetmap.org',
+            }),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 32767537,
+                'network': u'MY:TRG',
+                'shield_text': u'T11',
+            })
+
+    def test_q115_my_swk(self):
+        import dsl
+
+        z, x, y = (16, 52818, 32507)
+
+        self.generate_fixtures(
+            dsl.is_in('MY', z, x, y),
+            # https://www.openstreetmap.org/way/53546059
+            dsl.way(53546059, dsl.tile_diagonal(z, x, y), {
+                'highway': u'secondary',
+                'name': u'Jalan Akses Bau',
+                'oneway': u'no',
+                'ref': u'Q115',
+                'source': u'openstreetmap.org',
+            }),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 53546059,
+                'network': u'MY:SWK',
+                'shield_text': u'Q115',
+            })
+
+    def test_mbsa15_mysgrmunicipal(self):
+        import dsl
+
+        z, x, y = (16, 51252, 32209)
+
+        self.generate_fixtures(
+            dsl.is_in('MY', z, x, y),
+            # https://www.openstreetmap.org/way/24701880
+            dsl.way(24701880, dsl.tile_diagonal(z, x, y), {
+                'highway': u'primary',
+                'is_in': u'Shah Alam, Selangor, Malaysia',
+                'lanes': u'2',
+                'oneway': u'yes',
+                'ref': u'MBSA 15',
+                'source': u'openstreetmap.org',
+            }),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 24701880,
+                'network': u'MY:SGR:municipal',
+                'shield_text': u'MBSA 15',
```

Comment:
Drop the space in the `shield_text`?

Reply:
Also, this is more than 5 char total so either needs special handling client side or more server transform. Probably use 15 instead?

Reply:
From IRL conversation: `BSA15`